### DISCUSSION TITLE
Donations

### DISCUSSION THREAD

**Discussion by entangledloops** [libktx/ktx#406] [General]
Is there a link to provide donations to support this project somewhere? If there is, it isn't obvious,

**Comment by czyzby** [comment]
There isn't any. It has always been a passion project that I didn't really plan on monetizing. I think at this point both libGDX and KTX just get steady maintenance releases without any major new features. I'm not sure I would feel comfortable taking money for that, although admittedly it could motivate me to make the releases more often. I will think about it.

**Reply by czyzby** [reply]
I went ahead an set up an [Open Collective](https://opencollective.com/ktx) account for KTX. The project does not have any expenses per se, but I might just register a custom domain in the future if there's any interest. Thanks, @entangledloops.

**Reply by entangledloops** [reply]
Great, thanks!
